fetcher/nvd/json/nvd.go

```diff
@@ -360,12 +388,95 @@ func convertToModel(item *CveItem) (*models.CveDetail, error) {
 			Cpes:             cpes,
 			References:       refs,
 			Affects:          affects,
+			Certs:            certs,
 			PublishedDate:    publish,
 			LastModifiedDate: modified,
 		},
 	}, nil
 }
 
+func collectCertLinks(links []CertLink) (certs []models.Cert, err error) {
+	var proxyURL *url.URL
+	httpCilent := &http.Client{}
+	if c.Conf.HTTPProxy != "" {
+		if proxyURL, err = url.Parse(c.Conf.HTTPProxy); err != nil {
+			return nil, fmt.Errorf("failed to parse proxy url: %s", err)
+		}
+		httpCilent = &http.Client{Transport: &http.Transport{Proxy: http.ProxyURL(proxyURL)}}
+	}
+
+	reqChan := make(chan string, 10)
+	resChan := make(chan models.Cert, 10)
+	errChan := make(chan error, len(links))
+	defer close(reqChan)
+	defer close(resChan)
+	defer close(errChan)
+
+	go func() {
+		for _, ref := range links {
+			reqChan <- ref.Link
+		}
+	}()
+
+	concurrency := 3
+	tasks := util.GenWorkers(concurrency)
+	certs = []models.Cert{}
+
+	for _, l := range links {
+		tasks <- func() {
+			select {
+			case req := <-reqChan:
+				reqUrl, err := http.NewRequest("GET", req, nil)
```

Comment:
__[Golint]__ var reqUrl should be reqURL ([link](https://sider.review/gh/repos/54628930/pulls/129#issue-4304150))

<sub>posted by Sider</sub>